### PR TITLE
Proposed 1.8.1

### PR DESCRIPTION
**Pull request by manojsdoshi**
If merged this PR will:

close: #3997

### CODE REVIEW

**Comment by mDuo13** [comment]
Shouldn't there be a mention in the release notes about 9e877a9?

**Comment by manojsdoshi** [comment]
@mDuo13 [9e877a9](https://github.com/ripple/rippled/commit/9e877a929e46371bd74c1da08d66d727be77b79b) is tuning some RocksDB parameters which we might do this again. So I think it's ok to leave it out of the release notes

**Review by legleux — APPROVED** [review]

File: RELEASENOTES.md [hunk 1]
```diff
@@ -13,6 +13,36 @@ Have new ideas? Need help with setting up your node? Come visit us [here](https:
...
+- **More fields in the `validations` stream:** The `validations` subscription stream in the API now reports additional fields that were added to validation messages by the HardenedValidations amendment. These fields make it easier to detect misconfigurations such as multiple servers sharing a validation key pair. ([#3865](https://github.com/ripple/rippled/pull/3865))
+- **Reporting mode supports `validations` and `manifests` streams:** In the API it is now possible to connect to these streams when connected to a servers running in reporting. Previously, attempting to subscribe to these streams on a reporting server failed with the error `reportingUnsupported`. ([#3905](https://github.com/ripple/rippled/pull/3905))
+
+### Bug Fixes
+
+- **Clarify the safety of NetClock::time_point arithmetic**: * NetClock::rep is uint32_t and can be error-prone when   used with subtraction. * Fixes [#3656](https://github.com/ripple/rippled/pull/3656)
```
Comment (intelliot) [inline RELEASENOTES.md:35]:
```suggestion
- **Clarify the safety of NetClock::time_point arithmetic:** NetClock::rep is uint32_t and can be error-prone when used with subtraction. ([#3656](https://github.com/ripple/rippled/pull/3656))
```

**Review by intelliot — APPROVED** [review]
Minor formatting suggestion but not a blocker

**Pull request merged**